controllers/convert.js

```diff
@@ -14,10 +14,15 @@ module.exports = router.post('/',
     debug('Validated and rendered %s', res.locals.html);
 
     const model = new Model();
+
+    req.log('info', 'html-pdf-converter: Creating PDF');
```

Comment:
I'd probably make the debug messages be `debug` messages rather than `info`. Otherwise it's just going to be noise.